### PR TITLE
refactor/remove dead code

### PR DESCRIPTION
**Pull request by FredrikOseberg**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

File: app/hocs/withSuccessNotification.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function withSuccessNotification(
   message: Message,
   options: Object = {}
 ) {
-  const mapDisptchToProps = (dispatch, ownProps) => ({
+  const mapDispatchToProps = dispatch => ({
```
Comment (dvdschwrtz) [inline app/hocs/withSuccessNotification.js]: dispatch should have a type

**Comment by evgenyboxer** [comment]
@FredrikOseberg awesome stuff! could this be added as an eslint rule so that future dead code will fail the linter?

**Comment by FredrikOseberg** [comment]
@dvdschwrtz Thank you. I'll take care of those.
@evgenyboxer That is definitely possible, I'll update the PR with the rule specification.

**Comment by evgenyboxer** [comment]
@FredrikOseberg ?

**Comment by FredrikOseberg** [comment]
Apologies @evgenyboxer. I had a chat with @comountainclimber regarding this PR and we decided to put it on hold until v2 was released and include this in a patch at a later point. Since release my timetable has been filled, but I should be able to wrap this up tomorrow.

**Comment by comountainclimber** [comment]
This looks good to me @FredrikOseberg if we can address the missing type definitions that @dvdschwrtz identified we can merge this 👍

**Comment by FredrikOseberg** [comment]
@comountainclimber I took care of those in this commit: https://github.com/CityOfZion/neon-wallet/pull/1657/commits/574c82942033f152f16f5cafbc16d8562f8c7f7a 

I found another instance where it should be added in those files though, so I pushed those changes up.

**Pull request merged**